gamecreate.php

```diff
@@ -138,12 +138,6 @@
 			$input['bet'] = 5; 
 			$input['potType'] = 'Unranked';
 		}
-
-		// Only classic, no press can support fill with bots. 
-		if ( ($input['variantID'] != 1) || ($input['pressType'] != 'NoPress') )

```

Comment:
Are these deletions intentional? I don't see why they're relevant to a change that adds phase names between messages. Same for all the other changes to various php files in this PR.

Reply:
Oops, thanks for catching